src/models/SessionReturnRecord.ts

```diff
@@ -53,6 +62,8 @@ export class SessionReturnRecord {
 
     nameParts?: NamePart[];
 
+	postOfficeVisitDetails?: PostOfficeVisitDetails[];
```

Comment:
nit: indentation

Reply:
It looks good in the file, somehow not displaying well here.